src/mainwindow.cpp

```diff
@@ -830,9 +833,8 @@ void MainWindow::resetLibraryPlaylist() const
 
 void MainWindow::onSpotifyStatusChanged(const QString &status)
 {
-	if ((windowState() & Qt::WindowActive) > 0)
```

Comment:
I think the main point of this was to not grab attention when the window is minimized. Maybe `StatusMessage::error()` is better when the window isn't active? That way, you have to actively open the window to see if anything went wrong.